tests/srt.dora

```diff
@@ -0,0 +1,19 @@
+fun main() {
+    assert(0.0F.sortsAs(0.0F) == -1);    //wrong
+    assert(-0.0F.sortsAs(-0.0F) == 1);   //wrong
+    assert(1.0F.sortsAs(1.0F) == 1);     //wrong
+    assert(0.0F.sortsAs(1.0F) == -1);
+    assert(1.0F.sortsAs(0.0F) == 1);
+    assert(-0.0F.sortsAs(0.0F) == 1);    //wrong
+    assert(0.0F.sortsAs(-0.0F) == -1);   //wrong
+    assert((0.0F/0.0F).sortsAs(0.0F/0.0F) == -1); //wrong
+
+    assert(0.0.sortsAs(0.0) == -1);    //wrong
+    assert(-0.0.sortsAs(-0.0) == 1);   //wrong
+    assert(1.0.sortsAs(1.0) == 1);     //wrong
+    assert(0.0.sortsAs(1.0) == -1);
+    assert(1.0.sortsAs(0.0) == 1);
+    assert(-0.0.sortsAs(0.0) == 1);    //wrong
+    assert(0.0.sortsAs(-0.0) == -1);   //wrong
+    assert((0.0/0.0).sortsAs(0.0/0.0) == -1); //wrong
```

Comment:
Can't really see a rhyme or reason in which comparison returns the right or wrong result ...